grunt/tasks/migration.js

```diff
@@ -0,0 +1,151 @@
+module.exports = function(grunt) {
+
+  const Helpers = require('../helpers')(grunt);
+  const globs = require('globs');
+  const path = require('path');
+  const fs = require('fs-extra');
+  const _ = require('underscore');
+
+  function unix(path) {
+    return path.replace(/\\/g, '/');
+  }
+
+  function dressPathIndex(fileItem) {
+    return {
+      ...fileItem.item,
+      __index__: fileItem.index,
+      __path__: unix(fileItem.file.path)
+    };
+  }
+
+  function undressPathIndex(object) {
+    const clone = { ...object };
+    delete clone.__index__;
+    delete clone.__path__;
+    return clone;
+  }
+
+  grunt.registerTask('migration', 'Migrate from one version to another', function(mode) {
+    const next = this.async();
+    const buildConfig = Helpers.generateConfigData();
+    const fileNameIncludes = grunt.option('file');
+
+    (async function() {
+      const migrations = await import('adapt-migrations');
```

Comment:
you've not added this dependency to the package.json

Reply:
adapt-migrations moved to adaptlearning and published, added as devdependency

Reply:
https://www.npmjs.com/search?q=adapt-migrations

it doesn't look published

Reply:
https://www.npmjs.com/package/adapt-migrations

there we go